site/content/articles/external-container-registry-integration.adoc

```diff
@@ -0,0 +1,479 @@
+---
+date: "2019-12-01T19:15:24-06:00"
+title: "External Container Registry Integration with OpenShift"
```

Comment:
Can we change the title to "Integrating External Container Registries with OpenShift"